### PR TITLE
skipper-ingress: configure edit route placeholders

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -202,6 +202,9 @@ spec:
           - "-write-timeout-server={{ .ConfigItems.skipper_write_timeout_server }}"
           - '-default-filters-prepend={{ .ConfigItems.skipper_default_filters }}'
           - "-disabled-filters={{ .ConfigItems.skipper_disabled_filters }}"
+{{ if ne .ConfigItems.skipper_edit_route_placeholders "" }}
+          - '-edit-route={{ .ConfigItems.skipper_edit_route_placeholders }}'
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml:206]: Multiple values require https://github.com/zalando/skipper/issues/2106

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml:206]: will this fail in case `skipper_edit_route_placeholders` is not defined?

Comment (AlexanderYastrebov) [reply cluster/manifests/skipper/deployment.yaml:206]: I've added default empty value for it into config-defaults.yaml

**Comment by mikkeloscar** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**